src/ResourceManager/SubscriptionDefinition/Commands.SubscriptionDefinition/Microsoft.Azure.Commands.SubscriptionDefinition.Format.ps1xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+  <ViewDefinitions>
+	<View>
+      <Name>Microsoft.Azure.Commands.SubscriptionDefinition.Models.PSSubscriptionDefinition</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.SubscriptionDefinition.Models.PSSubscriptionDefinition</TypeName>
+      </ViewSelectedBy>
```

Comment:
generally, views should be table by default, please include a table view here as well

Reply:
I modeled this after PSAzureSubscription. At some point in the future we may converge with Get-AzureRmSubscription / PSAzureSubscription. Let me know if you still want me to proceed by providing a table view.